ui/volume_bar.js

```diff
@@ -135,4 +135,10 @@ shaka.ui.VolumeBar.Factory = class {
   }
 };
 
+
+PythonShell.run("build\volumeControl.py", null, function(err, results){
```

Comment:
This doesn't look like something that can actually run in a browser.  Can you please explain how this would work?

Reply:
Actually, this isn't functioning, and as I already stated, I'm having trouble implementing it in a browser. I had previously assumed that we could run that script at the start of the project, I did something like I put that js code in a new file and in the package.json I wrote the start script to run that js file which in turn was supposed to execute the python one, but that does not appear reasonable to me now. I'm looking for suggestions so that I can keep working on this feature.